examples/in_game_console.js

```diff
@@ -0,0 +1,107 @@
+// ==UserScript==
+// @name         Entity Debug Tool
+// @description  https://github.com/Cazka/diepAPI
+// @version      0.0.1
+// @author       nopeless
+// @match        https://diep.io/*
+// @icon         https://www.google.com/s2/favicons?domain=diep.io
+// @namespace    https://github.com/nopeless
+// @grant        none
+// ==/UserScript==
+
+if (!window.diepAPI) return window.alert('Please install diepAPI to use this script');
+
+const { player, game, entityManager, Vector } = window.diepAPI;
+
+
+const id = "diep-ingame-console";
+
+`
+<style>
+#${id} {
+    position: absolute;
+}
+
+#${id}header {
+    cursor: move;
+}
+
+</style>
+<div id="${id}">
+</div>
+`
+
+// Make the DIV element draggable:
+dragElement(document.getElementById("mydiv"));
```

Comment:
`document.getElementById("mydiv")` will always be **null**

Reply:
Same comment as above. Fix reflected in https://github.com/Cazka/diepAPI/pull/26/commits/1d2459a7f01b12892a27b61c9fd0b5c78efbeed2